<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRectDialog</class>
 <widget class="QDialog" name="AddRectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddRectDialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>216</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelX">
      <property name="text">
       <string>X Pos</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditX"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelY">
      <property name="text">
       <string>Y Pos</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditY"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelWidth">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelHeight">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditH"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelVel">
      <property name="text">
       <string>Velocity</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditVel"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButtonAdd">
      <property name="text">
       <string>Add Rectangle</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditW"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
